after-success.sh

```diff
@@ -4,10 +4,20 @@ set -e
 echo -e "Blackbaud - SKY UX Travis - After Success"
 
 function publish {
+
   echo -e "Publishing to NPM..."
-  npm publish --access public
+
+  # If the tag includes a '-' character, we can assume it's a prerelease version.
+  if [[ $TRAVIS_TAG =~ "-" ]]; then
+    echo -e "Publishing to NPM with tag 'next'.";
+    npm publish --access public --tag next
```

Comment:
Any objection to moving `next` or `latest` in to a variable, so that you only have to call the `echo` and `npm publish` commands once?  Something like (untested)...

```
npmTag = "next";

if [[ ! $TRAVIS_TAG =~  "-" ]]; then
  npmTag = "latest";
fi

echo -e "Publishing to NPM with tag '$npmTag'.";
npm public --access publish --tag $npmTag
```

Reply:
Yeah, I like that better.